packages/lodestar/src/sync/reqResp/reqResp.ts

```diff
@@ -280,10 +280,6 @@ export class BeaconReqRespHandler implements IReqRespHandler {
       );
       return;
     }
-    if (request.body.count > 1000) {
-      this.logger.warn(`Request id ${request.id} asked for ${request.body.count} blocks, just return 1000 maximum`);
-      request.body.count = 1000;
-    }
```

Comment:
why did you remove this?
You could change this so that "1000" (maxRequestedBlocks) be an ReqResp option.

Reply:
We have an identical check just below it with the correct number specified in the spec:
https://github.com/ChainSafe/lodestar/pull/1742/files#diff-7c5831b4dde5a221fa8a043078154dcd1d94033976ba1c25cc0e493ee3c7d156R284